code/index.html

```diff
@@ -3,28 +3,182 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
+    <title>Project 3: Business site - Balloon ride</title>
+    <link rel="stylesheet" href="./style.css" />
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Lobster&family=Roboto+Condensed:wght@300&display=swap');
+    </style>
     <!-- dont forget to add a css file and link it here! -->
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <!-- HERO IMAGE -->
+    <header>
+      <img src = "logo.png" alt= "Balloon Ride's logo">  
+    </header>
 
-    <!-- Signup form -->
-
-    <section class="accordion">
+    <div class= hero-container>
+      <div class="hero-image"></div>
+      <div class="hero-image-text"> Enter our contest and win a balloon ride!</div>       
```

Comment:
Since this is a headline and also the biggest one on your site it should perhaps be a `<h1>` instead 💫